go/consensus/tendermint/config/config.go

```diff
@@ -30,6 +30,9 @@ type Config struct {
 	// Transaction submission configuration.
 	Submission SubmissionConfig `yaml:"submission,omitempty"`
 
+	// Epoch at which to force-shutdown the node (in epochs).
```

Comment:
Not sure if we need to mention (e.g. in parentheses) that zero disarms shutdown (same for height).